database/migrations/2021_12_11_111524_create_books_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBooksTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('books', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('category_id')->references('id')->on('category')
+            ->cascadeOnUpdate()
+            ->cascadeOnDelete();
+            $table->string('name', 50)->default('Title!');
+            $table->integer('page_count');
+            $table->string('edition', 50);
```

Comment:
multiple places where 30 length transformed to 50. It was intended or it is copy paste?